man/tpm2_changepps.1.md

```diff
@@ -0,0 +1,42 @@
+% tpm2_changepps(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_changepps**(1) - Replaces the active platform primary seed with a new one
+generated off the TPM2 RNG.
+
+# SYNOPSIS
+
+**tpm2_changepps** [*ARGUMENT*]
+
+# DESCRIPTION
+
+**tpm2_changepps**(1) - Replaces the active platform primary seed with a new one
+generated off the TPM2 RNG. The Transient and Persistent objects under the
+platform hierarchy are lost whilst retaining the NV objects.
+
+# OPTIONS
+
+  * **ARGUMENT** the command line argument specifies the _AUTH_ for the platform
+  hierarchy.
```

Comment:
weird formatting here. `#options`  but no options...  I would make the password an option, just so its consistent between tools.

Reply:
True. Made it an option instead of an argument.